client/lib/create-selector/index.js

```diff
@@ -88,28 +87,29 @@ export default function createSelector(
 	getDependants = DEFAULT_GET_DEPENDANTS,
 	getCacheKey = DEFAULT_GET_CACHE_KEY
 ) {
-	const memoizedSelector = memoize( selector, getCacheKey );
-	let lastDependants;
+	const memo = new Map();
```

Comment:
It does feel like we'd run out of memory eventually if we never remove keys.

We have around 100+ usages of createSelector. Are we ok with at least having an LRU set to around a max of 1000 or so? I'm thinking most folks won't hit that many sites. Otherwise, should we look into a bigger pattern change like using reselect?

Reply:
depends on if we switch to making `perKey` an option or a separate function.